gcapi/client.py

```diff
@@ -898,22 +1091,51 @@ def add_cases_to_archive(
         *,
         archive: Union[str, gcapi.models.Archive],
         archive_items: list[SocketValueSetDescription],
-    ):
+    ) -> list[str]:
         """
         This function takes an archive slug or model and a list of archive item
         descriptions and creates the archive item to be used on the platform.
 
-        Parameters
-        ----------
-        archive
-            slug for the archive (e.g. `"i-am-an-archive"`). You can find this
-            readily in the URL you use to visit the archive page:
+        ??? tip "Re-using existing images"
```

Comment:
Is there not a way to deduplicate these comments so that this information is only in a single place, but correctly reused for the different functions in the docs?

Reply:
Right, it is just unfortunate if anything in this changes and someone forgets to update it in all places. But okay your logic makes sense.